recipes/libMobility/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set name = "libmobility" %}
+{% set version = "0.8.0" %}
+
+{% set cuda_compiler_version = cuda_compiler_version | default("None") %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/stochasticHydroTools/libMobility/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 259661175732bafad91ee0ef51d54da88adc31efa8ea7775e5452a2b492dee78
```

Comment:
Forgot to update your hashes?

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1264311&view=logs&j=df9408f7-e795-500f-143d-aa7735a0ed8b&t=47addc80-ccbf-566b-ee3a-66cdb8436937&l=912